templates/Menus/nas-main.html.twig

```diff
@@ -64,4 +64,4 @@
             </div>
         </li>
     </ul>
-</div>
+</div>
```

Comment:
Do you want to revert this change?

Reply:
Did I break something? Is it just a new line?

Reply:
Just the new line